torchci/lib/bot/retryBot.ts

```diff
@@ -0,0 +1,74 @@
+import { Probot } from "probot";
+
+const FAILURE_CONCLUSIONS = ["failure", "cancelled", "timed_out"];
+
+function retryBot(app: Probot): void {
+  app.on("workflow_run.completed", async (ctx) => {
+    const workflowName = ctx.payload.workflow_run.name;
+    const attemptNumber = ctx.payload.workflow_run.run_attempt;
+    if (
+      ctx.payload.workflow_run.conclusion === "success" ||
+      ctx.payload.workflow_run.head_branch !== "master" ||
+      attemptNumber > 1 ||
+      !["lint", "pull"].includes(workflowName)
```

Comment:
nit: could you add a comment explaining why we're using just these two workflows for now?